test/integration_test.py

```diff
@@ -1105,17 +1105,15 @@ def _test_repository_files(self, catalog: CatalogName):
             outer_file, inner_file = self._get_one_inner_file(catalog)
             source = self._source_spec(catalog, outer_file)
             file_url = furl(inner_file['url'])
-            self.assertEqual(file_url.path.segments[0], 'repository')
-            # FIXME: Use _check_endpoint() instead
-            #        https://github.com/DataBiosphere/azul/issues/7373
-            file_url.path.segments.insert(0, 'fetch')
-            response = self._get_url_unchecked(GET, file_url)
-            if response.status == 404:
-                response = json.loads(response.data)
+            if not file_url:
```

Comment:
Eliminate `not`, swap `if` and `else`.

Also, we shouldn't even call the `furl` constructor with None. The fact that it returns the same value as `furl('')` is insane. This conflates None with empty, which we're avoiding, as per contribution guidelines.